tests/integration/test_services_video_audio_extractor_transcriber.py

```
@@ -0,0 +1,1302 @@
+        mock_tag.track = "3"
+        mock_tag.disc = None
+        mock_tag.comment = None
+
+        mock_tinytag = MagicMock()
+        mock_tinytag.TinyTag.get.return_value = mock_tag
+
+        mock_mutagen = MagicMock()
+        mock_mutagen.File.side_effect = Exception("mutagen failed")
+
+        with patch.dict("sys.modules", {"mutagen": mock_mutagen, "tinytag": mock_tinytag}):
+            with patch("mutagen.File", mock_mutagen.File):
```

Ruff (0.15.9)

[warning] Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)